buildenv/jenkins/openjdk_tests

```diff
@@ -170,30 +170,31 @@ timestamps{
                 LABEL = params.LABEL
             } else {
                 LABEL = PLATFORM_MAP[params.PLATFORM]["LABEL"]
-                    if (params.BUILD_LIST.contains("perf")) {
-                        def perfLabel = LABEL.minus("ci.role.test&&").concat("&&ci.role.perf")
-                        if (areNodesWithLabelOnline(perfLabel)) {
+                if (params.BUILD_LIST.contains("perf")) {
+                    def perfLabel = LABEL.minus("ci.role.test&&").concat("&&ci.role.perf")
+                    if (areNodesWithLabelOnline(perfLabel)) {
                         LABEL = perfLabel
-                        }
                     }
                 }
+                if (params.TEST_FLAG == "FIPS") {
+                    LABEL = LABEL.minus("ci.role.test&&").concat("&&ci.role.test.fips")
+                }
+                // Temporarily exclude ubuntu 22 machines for criu sanity.external pipeline
+                if (JOB_NAME.contains ("sanity.external") && JOB_NAME.contains("_criu")) {
+                    LABEL += "&&!sw.os.ubuntu.22"
+                }
```

Comment:
This is no longer needed as CRIU is moved out of sanity.external. @LongyuZhang could you remove this in a separate PR? Thanks.